pandas/tests/test_algos.py

```diff
@@ -1222,7 +1222,7 @@ def test_group_var_constant(self):
 class TestGroupVarFloat64(GroupVarTestMixin):
     __test__ = True
 
-    algo = libgroupby.group_var_float64
+    algo = staticmethod(libgroupby.group_var_float64)
```

Comment:
The reason this is needed is that algo is a class member (which is subsequently called using `self.algo(...)`.

Until now group_var_float64 was a cython builtin function which, for some reason, was not bound to self when called, so it was basically treated as a staticmethod.

With this PR, group_var_float64 (and all the other functions moved from tempita to a fused types implementation) is a cython fused type function, which gets bound to self when called using `self.algo`, so self is passed as the first argument which shifts all the arguments and causes problems with the test.

The solve this, I wrapped group_var_float64 as a staticmethod in the test.

Reply:
should prob just move these to regular test functions and parameterize as needed to avoid this; but that's for a future followup